theme/templates/base.html

```diff
@@ -5,7 +5,7 @@
 <head>
   <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>{% block title %}Tryton - World's Most Stable OpenSource ERP System{% endblock title %}</title>
+  <title>{% block title %}Tryton - Business solution modular, scalable and reliable{% endblock title %}</title>
```

Comment:
I think 

`Tryton - Modular, Scalable and Reliable Business Solution` would be better